rp2040-hal/CHANGELOG.md

```diff
@@ -8,24 +8,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
-- implement `rng_core::RngCore` for `RingOscillator`
+
+- None
 
 ### Changed
-- Modified PIO API for better ergonomics
 
-## [0.3.0] - 2021-09-20
+- None
+
+## [0.3.0] - 2021-12-19
 
 ### Added
-- peripheral drivers: Timer(counter) with EH traits, USB, PIO
-- examples: Watchdog, GPIO, ADC, i2c, rtic blinky, usb (serial echo, mouse)
-- docs: Watchdog
-- bsps: adafruit feather
-- bsp examples: sparkfun pro micro PIO RGB rainbow LED
+- A README!
+- Implementation of the `critical-section` API
+- DMA support
+- Embedded HAL 1.0 Alpha support
```

Comment:
Each alpha is a breaking release, should specify alpha.6 here so users know which version to expect